### PR TITLE
css-brochure-form | Daniel Hong

### PR DESCRIPTION
**Pull request by daniel-sj-hong**
Screenshot 2021-05-06 095629

### CODE REVIEW

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      <div id="name-input">
+        <input type="text" placeholder="Sally Smith">
+      </div>
+
+      <h2>WHERE ARE YOU HIRING?</h2>
+      <div id="location-input">
+        <input type="text" placeholder="City, Country">
+      </div>
+
+      <h2>HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</h2>
```
Comment (codingcodymiller) [inline css-brochure-form/index.html:32]: I would make this a `label` element rather than a `h2`

File: css-brochure-form/index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      <div id="email-input">
+        <input type="text" placeholder="sally@company.co">
+      </div>
+
+      <h2>NAME</h2>
+      <div id="name-input">
+        <input type="text" placeholder="Sally Smith">
+      </div>
+
+      <h2>WHERE ARE YOU HIRING?</h2>
```
Comment (codingcodymiller) [inline css-brochure-form/index.html:27]: I would make this a `label` element rather than a `h2`

File: css-brochure-form/index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  <div id="container">
+
+    <h1>LEARN MORE</h1>
+    <form>
+      <h2>COMPANY EMAIL</h2>
+      <div id="email-input">
+        <input type="text" placeholder="sally@company.co">
+      </div>
+
+      <h2>NAME</h2>
```
Comment (codingcodymiller) [inline css-brochure-form/index.html:22]: I would make this a `label` element rather than a `h2`

File: css-brochure-form/index.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div id="container">
+
+    <h1>LEARN MORE</h1>
+    <form>
+      <h2>COMPANY EMAIL</h2>
```
Comment (codingcodymiller) [inline css-brochure-form/index.html:17]: I would make this a `label` element rather than a `h2`

File: css-brochure-form/styles.css [hunk 5]
```diff
@@ -0,0 +1,75 @@
+#container {
+  width: 25%;
+  margin: auto;
+}
+
+h1 {
+  width: 50%;
+  margin: auto;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:8]: These can be removed, `text-align` will take care of this for you 👍

File: css-brochure-form/styles.css [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+  text-align: center;
+  color: #ec605a;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  src: url('BrandonGrotesque-Medium.woff2') format('woff2');
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:14]: These are unnecessary, they are only needed in the `@font-face` rule, which you don't need to worry about for now 👍

File: css-brochure-form/styles.css [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+  padding: 15px;
+}
+
+h2 {
+  margin-bottom: 1px;
+  margin-left: 8px;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  src: url('BrandonGrotesque-Medium.woff2') format('woff2');
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:25]: These are unnecessary, they are only needed in the `@font-face` rule, which you don't need to worry about for now 👍

File: css-brochure-form/styles.css [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+  font-size: 12px;
+}
+
+input {
+  width: 100%;
+  height: 30px;
+  border-radius: 5px;
+  border-color: #ced0d0;
+  border-style: groove;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:34]: This should be `solid`

File: css-brochure-form/styles.css [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+  padding-left: 7px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-right: 10px;
+  font-family: 'AtlasGrotesk', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  src: url('AtlasGrotesk-Light-Web.woff2') format('woff2');
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:42]: These are unnecessary, they are only needed in the `@font-face` rule, which you don't need to worry about for now 👍

File: css-brochure-form/styles.css [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+  width: 100%;
+  height: 30px;
+  border-radius: 5px;
+  border-color: #ced0d0;
+  border-style: groove;
+  background-color: #f3f3f3;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  src: url('BrandonGrotesque-Regular.woff2') format('woff2');
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:55]: These are unnecessary, they are only needed in the `@font-face` rule, which you don't need to worry about for now 👍

File: css-brochure-form/styles.css [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+  padding: 5px;
+  margin-top: 10px;
+  border-radius: 30px;
+  width: 90px;
+  height: 30px;
+  border: none;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  src: url('BrandonGrotesque-Medium.woff2') format('woff2');
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:74]: These are unnecessary, they are only needed in the `@font-face` rule, which you don't need to worry about for now 👍

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start here, man! A few notes for you:

**Comment by daniel-sj-hong** [comment]
Screenshot 2021-05-06 172608

File: css-brochure-form/styles.css (hunk 8 above)
Comment (codingcodymiller) [reply css-brochure-form/styles.css:26]: This still needs to be addressed

File: css-brochure-form/index.html [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+      <div id="name-input">
+        <input type="text" placeholder="Sally Smith">
+      </div>
+
+      <label>WHERE ARE YOU HIRING?</label>
+      <div id="location-input">
+        <input type="text" placeholder="City, Country">
+      </div>
+
+      <label>HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label>
```
Comment (codingcodymiller) [inline css-brochure-form/index.html:32]: You should apply a `for` attribute to each of your `label` elements that matches to its form control element.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
This is looking much better! A few more notes for you:

**Review by codingcodymiller — APPROVED** [review]
Nice work, dude!!

**Pull request merged**